pkg/capacity/capacity.go

```diff
@@ -228,24 +232,28 @@ var _ metrics.StableCollector = &Controller{}
 
 // Run is a main Controller handler
 func (c *Controller) Run(ctx context.Context, threadiness int) {
-	klog.Info("Starting Capacity Controller")
+	logger := c.logger
+
+	logger.Info("starting controller")
 	defer c.queue.ShutDown()
 
-	c.prepare(ctx)
+	c.prepare(ctx, logger)
 	for i := 0; i < threadiness; i++ {
 		go wait.UntilWithContext(ctx, func(ctx context.Context) {
-			c.runWorker(ctx)
+			c.runWorker(ctx, logger)
 		}, time.Second)
 	}
 
-	go wait.UntilWithContext(ctx, func(ctx context.Context) { c.pollCapacities() }, c.pollPeriod)
+	go wait.UntilWithContext(ctx, func(ctx context.Context) { c.pollCapacities(logger) }, c.pollPeriod)
 
-	klog.Info("Started Capacity Controller")
+	logger.Info("started controller")
```

Comment:
What's the reason for starting an info log message with lower case?

Reply:
It's not a full sentence, therefore initial capital letter looked odd. I don't know whether there is some guidance on this.

It's also consistent with error messages. For those the official guidance is to start with lower case because the error might get wrapped.

Reply:
I'd like to see if the structured log KEP has any official guidance on this.  The official guidance for error message is only referring to the returned error message, not error messages in the logs.

Reply:
The logr example uses lower case, incidentally also with a "starting" message:
https://github.com/go-logr/logr#typical-usage

But the Kubernetes documentation says "Start from a capital letter": https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#remove-string-formatting-from-log-message

It also has some other recommendations. Once we agree to go further with this and the klog PR is merged, I'll revisit the log messages and update them accordingly.